spec/features/user_index_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'rails_helper'
+
+RSpec.describe 'User index', type: :feature do
+  before(:each) do
```

Comment:
Please add a test that redirects to the user's show page when the user clicks on a user.

Reply:
Great, done. Thanks!